src/pages/student-home/StudentHome.tsx

```
@@ -7,9 +7,11 @@ import { useModalContext } from '~/context/modal-context'
 import UserStepsWrapper from '~/components/user-steps-wrapper/UserStepsWrapper'
 import FindBlock from '~/components/find-block/FindBlock'
 import Faq from '~/containers/student-home-page/faq/Faq'
+import PopularCategories from '~/containers/find-offer/popular-categories/PopularCategories'
 
 import { translationKey } from '~/components/find-block/find-tutor-constants'
 import { HowItWorksBlock } from '~/components/how-it-works/HowItWorksBlock'
+import { Stack } from '@mui/material'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Stack'. Consider to delete it or rename it.

'Stack' is defined here:


(lint/suspicious/noRedeclare)